src/main/native/include/cscore.h

```diff
@@ -1,12 +1,12 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CSCORE_H_
-#define CSCORE_H_
+#ifndef INCLUDE_CSCORE_H_
+#define INCLUDE_CSCORE_H_
```

Comment:
I still need to implement the new header guard pattern in wpiformat. There are some headers in the source instead of the include directory because they shouldn't be published, so I can't split the path to the right of "include". The problem is we really need to teach wpiformat about our special directory structure somehow so it knows where to cut it. Perhaps we can give cpplint.py a list of include roots for possible include guard truncation in the same way the repo root is used.